ci/env/install-bazel.sh

```diff
@@ -98,24 +98,6 @@ bazel --version
 # clear bazelrc
 echo > ~/.bazelrc
 
-if [[ "${TRAVIS-}" == true ]]; then
-  echo "build --config=ci-travis" >> ~/.bazelrc
-
-  # If we are in Travis, most of the compilation result will be cached.
-  # This means we are I/O bounded. By default, Bazel set the number of concurrent
-  # jobs to the the number cores on the machine, which are not efficient for
-  # network bounded cache downloading workload. Therefore we increase the number
-  # of jobs to 50
```

Comment:
I don't think this makes much sense btw, bazel will self-limit based on cpu and mem anyways..